readme.md

```diff
@@ -38,63 +38,65 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** Forking is a request to clone and registers under your username.
 
 ---
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
-* **Your Answer:**
+* **Your Answer:** Created a package.json file with information about the author, repo, etc.
 
 ---
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
-* **Your Answer:**
+* **Your Answer:** name = project name
+scripts = runs tests
+license = software license
 
 ---
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
-* **Your Answer:**
+* **Your Answer:** It tells the repo what files to ignore.
```

Comment:
And, a "dot-file" is a hidden file, typically one that deals with configuration.